schema/bundle.schema.json

```diff
@@ -308,6 +308,10 @@
                             "type": "string"
                         }
                     }
+                },
+                "apply-to":{                    
+                    "description": "An optional exhaustive list of actions handling this parameter",
+                    "type":"array"
```

Comment:
This should probably also specify that the items can only be strings, given the purpose is a list of actions. Anything else should be invalid I think.

```json
"items": {
  "type": "string"
}
```